js_modules/dagster-ui/packages/ui-core/src/instance/VirtualizedInstanceConcurrencyTable.tsx

```diff
@@ -114,11 +85,15 @@ const ConcurrencyRow = ({
   const {data} = queryResult;
   const limit = data?.instance.concurrencyLimit;
 
+  const path = `/deployment/concurrency/${concurrencyKey}`;
```

Comment:
I'm not sure what the format of concurrencyKey is, but it may be worth doing a `encodeURIComponent(concurrencyKey)`  or testing what happens when it contains spaces, etc. Link may handle it but I'm not sure!